### PR TITLE
Make the categories listing look better by resizing the images on hover

### LINKED ISSUE #84: Make the categories listing look better by resizing the images on hover
Also don't fade the colors away when hovering, maybe add a filter through css to make it look grayscale. And work with Hazim to incorporate his changes for the carousel constraining as well.

### PR DESCRIPTION
**Pull request by karenchehade**

### CODE REVIEW

**Review by hsank96 — APPROVED** [review]

File: src/components/Home/categoryGrid.jsx [hunk 1]
```diff
@@ -3,14 +3,29 @@ import CategoryCard from "./categoryCard";
 import Categories from "./Categories";
 import { Container } from "react-bootstrap";
 import { Link } from "react-router-dom";
+import "./home.css";
 
 function CategoryGrid() {
   return (
     <Container>
-      <div className="row row-cols-xl-3 row-cols-md-3 row-cols-sm-2">
+      <svg>
```
Comment (wisammechano) [inline src/components/Home/categoryGrid.jsx:11]: What is this SVG? it doesn't show up on the website

File: src/components/Home/home.css [hunk 2]
```diff
@@ -8,24 +8,34 @@
...
   padding-top: 4%;
   height: 80%;
-  width: 30%;
   margin-top: 15%;
 }
-.categoryCard :hover {
-  opacity: 0.7;
-  transform: scale(1.2);
+
+.card-body {
```
Comment (wisammechano) [inline src/components/Home/home.css:20]: adding a `cursor: pointer;` would be cool

File: src/components/Home/categoryGrid.jsx (hunk 1 above)
Comment (wisammechano) [reply src/components/Home/categoryGrid.jsx:11]: will keep this for now, and we can delete later.

File: src/components/Home/home.css (hunk 2 above)
Comment (wisammechano) [reply src/components/Home/home.css:20]: added to category-card

**Review by wisammechano — APPROVED** [review]

**Pull request merged**